### PR TITLE
create index.html, add simple html markup

### PR DESCRIPTION
**Pull request by JessicaLoers**
- add a super fancy headline
- something

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Super todo app</title>
+  </head>
+  <body>
+    <h1>Hello Friends.</h1>
```
Comment (JessicaLoers) [inline index.html]: Use a more descriptive Headline

**Review by JessicaLoers — COMMENTED** [review]
LGTM ✨
Add a tiny comment regarding the headline.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Super todo app</title>
+  </head>
+  <body>
+    <h1>Welcome to the super duper to do app</h1>
```
Comment (nadine-grantz) [inline index.html:9]: well done

**Review by nadine-grantz — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (JessicaLoers) [reply index.html:9]: Thank ✨

**Pull request merged**